x/lscosmos/keeper/abci.go

```diff
@@ -33,8 +33,7 @@ func (k Keeper) DoDelegate(ctx sdk.Context) error {
 	hostAccounts := k.GetHostAccounts(ctx)
 
 	delegatableAmount := delegationState.HostDelegationAccountBalance.AmountOf(hostChainParams.BaseDenom)
-	allowlistedValidators := k.GetAllowListedValidators(ctx)
-	if !delegatableAmount.GT(sdk.NewInt(int64(len(allowlistedValidators.AllowListedValidators)))) {
+	if !delegatableAmount.IsPositive() {
```

Comment:
why this? i thought we needed the min value otherwise will break weight time delegation

Reply:
pushed change here